azkaban-common/src/test/java/azkaban/executor/mail/DefaultMailCreatorTest.java

```diff
@@ -0,0 +1,121 @@
+package azkaban.executor.mail;
+
+import azkaban.executor.ExecutableFlow;
+import azkaban.executor.ExecutionOptions;
+import azkaban.executor.Status;
+import azkaban.flow.Flow;
+import azkaban.flow.Node;
+import azkaban.project.Project;
+import azkaban.utils.EmailMessage;
+import azkaban.utils.Utils;
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.io.IOUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.util.TimeZone;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class DefaultMailCreatorTest {
+
+  private DefaultMailCreator mailCreator;
+
+  private ExecutableFlow executableFlow;
+  private Flow flow;
+  private Project project;
+  private ExecutionOptions options;
+  private EmailMessage message;
+  private String azkabanName;
+  private String scheme;
+  private String clientHostname;
+  private String clientPortNumber;
+  private TimeZone defaultTz;
+
+  @Before
+  public void setUp() throws Exception {
+    defaultTz = TimeZone.getDefault();
+    assertNotNull(defaultTz);
+    // EEST
+    TimeZone.setDefault(TimeZone.getTimeZone("Europe/Helsinki"));
+    Utils.setCurrentTimeMillisFixed(1468745661608L);
```

Comment:
Does it make sense to use a constant and add a comment to explain what it represents?